File: salary.java
import java.util.*;

class salary
{public static void main (String[] args) throws java.lang.Exception
	{try{
	    Scanner s =new Scanner(System.in);
	    int t = 0;
	    if(s.hasNextInt()){
	    	t=s.nextInt();
	    }
	     while(t>0){
	     	int n =0; 
	     	if(s.hasNextInt()){
	     		n=s.nextInt();
	     	}
	        int count=0,min=0;
	        int a[]=new int[n];
	        for(int i=0;i<n;i++){
	            a[i]=s.nextInt();
	            if(a[i]<a[min])
	                min=i;}
	        for(int i=0;i<n;i++)
	            count+=a[i]-a[min];
	        System.out.println(count);
	        t--;
	    }
	}catch(Exception e){}
	}
}

File: code/online_challenges/src/codechef/SALARY/SALARY.c
#include<stdio.h>

int main() 
{
    int t;
    scanf("%d", & t);
    int T[t];
    int i, j;
    for (i = 0; i < t; ++i) 
    {
        int n;
        scanf("%d", & n);
        int A[n];
        int min = 32767;
        for (j = 0; j < n; ++j) 
        {
            scanf("%d", & A[j]);
            if (A[j] < min) 
            {
                min = A[j];
            }
        }
        int count = 0;
        for (j = 0; j < n; ++j) 
        {
            count += (A[j] - min);
        }
        T[i] = count
    }

    for (i = 0; i < t; ++i) 
    {
        printf("%d\n", T[i]);
    }
    return 0;
}
